openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -597,6 +618,256 @@ Location: https://client.example.net/cb?
   &error_description=Unsupported%20response_type%20value
 ```
 
+# Interactive Authorization Endpoint
+
+This is an extension of the traditional Authorization Endpoint defined in [@!RFC6749], enabling complex authentication and authorization flows where interaction occurs directly with the Wallet rather than being intermediated by a browser. 
+A primary use case is requiring the Presentation of a Credential as a prerequisite for issuing a new Credential.
+Support for the Interactive Authorization Endpoint is OPTIONAL.
+
+The Authorization Server indicates support for interactive authorization by publishing the `interactive_authorization_endpoint` parameter in its Authorization Server Metadata. In this case, the Wallet SHOULD use this endpoint to obtain authorization.
+
+Note: This mechanism can only be used for interactions with the same Wallet that started the issuance process.
+
+The following figure illustrates a flow using the Interactive Authorization Endpoint, where the Authorization Server requests a Presentation (of another Credential) from the Wallet as part of the authorization process to issue a Credential to that Wallet. The exact deployment model of the OpenID4VP Verifier in relation to the Authorization Server is out of scope of this specification. It can be integrated into the Authorization Server or a separate component, in which case backchannel communication between the Verifier and Authorization Server would need to happen (not shown here).
+
+
+!---
+~~~ ascii-art
```

Comment:
This sequence diagram is helpful but it still misses the overal context of the protocol flow IMO, for example how does a wallet know to hit the interactive authorization endpoint vs the conventional authorization endpoint OR the par endpoint?

Reply:
This is covered in the introduction - "In this case [the endpoint being present], the Wallet SHOULD use this endpoint to obtain authorization.".